README.md

```diff
@@ -1,58 +1,48 @@
-# Open棟梁(OpenTouryo)
-”Open棟梁”は、長年の.NETアプリケーション開発実績にて蓄積したノウハウに基づき開発した.NET用アプリケーション フレームワークです。
-(”OpenTouryo” , is an application framework for .NET which was developed using the accumulated know-how with a long track record in .NET application development.)
+# OpenTouryo
+"OpenTouryo", is an application framework for .NET which was developed using the accumulated know-how with a long track record in .NET application development.
 
-## 開発／動作環境(develop/run-time environment)
-このリポジトリのプログラムは、サポートされる最も古い開発／実行環境を使用して開発しています。 (I am developing the program of this repository, using the oldest supported develop/run-time environment.)
-- 開発環境 (develop environment)
-    - Visual Studio 2010
-- 実行環境 (run-time environment)
-    - .NET Framework 3.5 SP1
+## Develop/Run-time environment
+The program in this repository is developed, using the oldest supported develop/run-time environment.
 
-Visual Studio 2015 など、新しい開発環境を使用する場合は、[テンプレート・ベース](https://github.com/OpenTouryoProject/OpenTouryoTemplates)を参照してください。 (If you want to use the latest develop environment (e.g. Visual Studio 2015), please refer [Templates base](https://github.com/OpenTouryoProject/OpenTouryoTemplates).)
+- Develop environment
+  - Visual Studio 2015
+- Run-time environment
+  - .NET Framework 4.6
 
-## 概要(summary)
-以下のファイルを参照してください。
-(Please refer to the following files.)
+If you want to use the latest development environment (e.g. Visual Studio 2015), please refer [Templates base](https://github.com/OpenTouryoProject/OpenTouryoTemplatesV2).
 
+## Summary
+Please refer to the following files.
  - [Introduction.pdf](https://github.com/OpenTouryoProject/OpenTouryoDocuments/blob/master/documents/0_Introduction/ja-JP/Introduction.pdf)
  - [Introduction.pptx](https://github.com/OpenTouryoProject/OpenTouryoDocuments/blob/master/documents/0_Introduction/ja-JP/Introduction.pptx)
  - [Introduction_EN.pdf](https://github.com/OpenTouryoProject/OpenTouryoDocuments/blob/master/documents/0_Introduction/en/Introduction_EN.pdf)
  - [Introduction_EN.pptx](https://github.com/OpenTouryoProject/OpenTouryoDocuments/blob/master/documents/0_Introduction/en/Introduction_EN.pptx)
  - [Functional_list.xlsx](https://github.com/OpenTouryoProject/OpenTouryoDocuments/blob/master/documents/0_Introduction/ja-JP/Functional_list.xlsx)
 
-ドキュメント類は[OpenTouryoProject/OpenTouryoDocuments](https://github.com/OpenTouryoProject/OpenTouryoDocuments)リポジトリに格納されています。
-(documents are located in the [OpenTouryoProject/OpenTouryoDocuments](https://github.com/OpenTouryoProject/OpenTouryoDocuments) repository.)
+Documents are located in the [OpenTouryoProject/OpenTouryoDocuments](https://github.com/OpenTouryoProject/OpenTouryoDocuments) repository.
```

Comment:
@daisukenishino 

I think that this sentence is unnecessary.
(Same sentence exists in *Details* section.)